### PR TITLE
Add Deako integration documentation

### PR DESCRIPTION
**Pull request by Balake**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121132
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5633
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added detailed instructions for integrating Deako Smart Lighting into Home Assistant, enabling control of Deako devices through the platform.
  - Introduced support for various Deako smart devices, including functionality for device communication and setup.
  - Specified that certain features, such as groups and scenes, are currently unsupported.

### CODE REVIEW

File: source/_integrations/deako.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Deako Smart Lighting
+description: Instructions on how to integrate Deako Smart Lighting into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2024.7.1
```
Comment (joostlek) [inline source/_integrations/deako.markdown]:
```suggestion
ha_release: 2024.10
```

File: source/_integrations/deako.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2024.10
+ha_domain: deako
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+The `deako` platform allows you to control your [Deako](https://deako.com) devices from Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown]:
```suggestion
The **Deako Smart Lighting** integration allows you to control your [Deako](https://deako.com) devices from Home Assistant.
```

File: source/_integrations/deako.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: Deako Smart Lighting
+description: Instructions on how to integrate Deako Smart Lighting into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2024.10
+ha_domain: deako
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown:8]:
```suggestion
ha_domain: deako
ha_config_flow: true
```

File: source/_integrations/deako.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ha_platforms:
+  - light
+ha_integration_type: integration
+---
+
+The `deako` platform allows you to control your [Deako](https://deako.com) devices from Home Assistant.
+
+In order for this integration to work, your devices must be listed **online**, viewable in the Deako app.
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown]:
```suggestion
## Prerequisites

For this integration to work, your devices must be listed **online**, viewable in the Deako app.
```

File: source/_integrations/deako.markdown [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown:23]:
```suggestion

{% include integrations/config_flow.md %}

```

File: source/_integrations/deako.markdown [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Current devices supported:
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown]:
```suggestion
## Supported devices
```

File: source/_integrations/deako.markdown [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+title: Deako Smart Lighting
+description: Instructions on how to integrate Deako Smart Lighting into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: 2024.10
+ha_domain: deako
+ha_platforms:
+  - light
+ha_integration_type: integration
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown:17]:
```suggestion
ha_codeowners:
  - '@sebirdman'
  - '@balake'
  - '@deakolights'
ha_zeroconf: true
ha_integration_type: integration
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Add codeowners

File: source/_integrations/deako.markdown (hunk 4 above)
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown]: Do I need the app just for the commissioning/onboarding of the devices? Can I uninstall the app once the devices are in Home Assistant? Or do they stop working in HA if I uninstall the app?

Comment (Balake) [reply source/_integrations/deako.markdown]: Good question. You can uninstall the app once devices have been onboarded. I will add that.

File: source/_integrations/deako.markdown [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+## Supported devices
+
+- all smart Deako devices
+
+Features not currently supported:
+
+- Deako groups
+- Deako scenes
+
+This integration uses the [`pydeako`](https://pypi.org/project/pydeako/) library to connect to and interact with devices. One device is selected as the bridge over the local network to the Deako BLE mesh network, allowing for discovery of all devices and control.
```
Comment (c0ffeeca7) [inline source/_integrations/deako.markdown]:
```suggestion
One device is selected as the bridge over the local network to the Deako BLE mesh network, allowing for discovery of all devices and control.
```
This note about the library used, is this needed? what would I do with this as an end user?

Comment (Balake) [reply source/_integrations/deako.markdown]: It's not needed. I'll remove it.

Comment (Balake) [reply source/_integrations/deako.markdown]: I need to remove the part about manual setup also as that's no longer applicable

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Balake :+1:

**Pull request merged**